### DISCUSSION TITLE
performing (the same) transform to all submaps in one subject, and storing them

### DISCUSSION THREAD

**Discussion by madssakre** [TorchIO-project/torchio#1169] [Q&A] [answered]
If a subject looks like
```
subject = tio.Subject(
        A=tio.ScalarImage('A.nii.gz',),
        B=tio.ScalarImage('B.nii.gz',),
        )
```
where A and B has the same matrix size.
How do I perform the transformations to both A and B and store them either as a concatenated matrix in one nii.gz or in separate files?

**Comment by fepegar** [comment, marked as answer]
Hi @madssakre. Please take a look at the [tutorials](https://github.com/fepegar/torchio/blob/main/tutorials/README.md).

The same transformation will be applied to both images. You can simply save them like `subject.A.save('image_a.nii.gz')`.
